state-chain/chains/src/sol/sol_tx_core.rs

```diff
@@ -772,6 +772,61 @@ impl From<Hash> for SolHash {
 	}
 }
 
+#[derive(Encode, Decode, TypeInfo, Serialize, Deserialize, Debug, Clone)]
+pub struct CcmExtraAddress {
+	pub_key: Pubkey,
+	writable: bool,
+}
+
+impl From<CcmExtraAddress> for AccountMeta {
+	fn from(from: CcmExtraAddress) -> Self {
+		match from.writable {
+			true => AccountMeta::new(from.pub_key, false),
+			false => AccountMeta::new_readonly(from.pub_key, false),
+		}
+	}
+}
+
+#[derive(Encode, Decode, TypeInfo, Serialize, Deserialize, Debug, Clone)]
+pub struct CcmExtraAccounts {
+	pub cf_receiver: CcmExtraAddress,
+	pub remaining_accounts: Vec<CcmExtraAddress>,
+}
+
+impl CcmExtraAccounts {
+	pub fn remaining_account_metas(self) -> Vec<AccountMeta> {
+		self.remaining_accounts.into_iter().map(|acc| acc.into()).collect::<Vec<_>>()
```

Comment:
AccountMeta has an extra parameter, `isSigner` that is not passed as part of the ccm accounts. Will this default that value to false?

Reply:
Yes. in the conversion trait impl:
```impl From<CcmAddress> for AccountMeta {```

the is_signer is hard-coded to be false